src/Core/Constants.cs

```diff
@@ -3,6 +3,10 @@
     public static class Constants
     {
         public const int BypassFiltersEventId = 12482444;
+
+        // File size limits - give 1 MB extra for cushion
+        public const long FileSize101mb = 101L * 1024L * 1024L;
+        public const long FileSize501mb = 501L * 1024L * 1024L;
```

Comment:
I would suggest adding a comment here that we need to update the Nginx config as well. Should help avoiding this issue in the future.